opus/import/obs_cassini_common.py

```
@@ -109,12 +126,186 @@
+# W/MT3/BL1
+
 class ObsCassiniCommon(ObsBase):
 
     ################################################################################
     # HELPER FUNCTIONS USED BY CASSINI INSTRUMENTS
     ################################################################################
 
+    # Equinox: 2009-08-11T01:40:08.914
+    # After this time, north side of the ring is lit.
+    # Before this time, south side of the ring is lit.
+    def _is_ring_north_side_lit(self):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `_is_ring_north_side_lit`

(ANN202)